src/Xamarin.Android.Build.Tasks/Tasks/NdkUtils.cs

```diff
@@ -14,84 +14,86 @@
 
 namespace Xamarin.Android.Tasks
 {
-	public static class NdkUtil {
+	public static class NdkUtil
+	{
+		static bool usingNewNDK;
+		static bool initialized;
+
+		public static bool UsingNewNDK => usingNewNDK;
+
+		public static void Init (string ndkPath)
```

Comment:
This feels like it could break if the NDK is updated/replaced while the IDE has an XA project loaded.

I don't know how likely that is to happen. :-)